config/hydroshare-config.yaml

```diff
@@ -2,6 +2,7 @@
 HS_PATH: /home/hydro/hydroshare
 HOST_SSL_DIR: /home/hydro/hs-certs
 HS_DATABASE: pg.development.sql
+HS_LOG_FILES: /home/hydro/log
```

Comment:
In the same way that `HS_PATH` is user definable based on the local environment, so is `HS_LOG_FILES`.

This could just as easily be set to `/var/log/hydroshare` or `/home/$USER/mylogs`.

I'll default it to exist within the top level of the hydroshare repository as `/home/hydro/hydroshare/log` with an addition to the `.gitignore` file to ignore the **log** directory if it exists. The user can then adjust the `hydrosphere-config.yaml` file locally to fit their needs.